Upstream: No, disables parts of test suite
Source: William Orr <will@worrbase.com>
Reason: test/simple/test-tls-sni-server-client binds to 0.0.0.0 -- don't do that

--- node-orig/test/simple/test-tls-sni-server-client.js	2012-08-22 14:03:20.000000000 -0400
+++ node-v0.8.8/test/simple/test-tls-sni-server-client.js	2012-08-29 18:38:58.606297829 -0400
@@ -89,7 +89,7 @@
 server.addContext('a.example.com', SNIContexts['a.example.com']);
 server.addContext('*.test.com', SNIContexts['asterisk.test.com']);
 
-server.listen(serverPort, startTest);
+server.listen(serverPort, "127.0.0.1", startTest);
 
 function startTest() {
   function connectClient(options, callback) {
